src/components/List/List.test.js

```diff
@@ -1,8 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import List from './List';
+import List, { timeDifference } from './List';
 import { BrowserRouter } from 'react-router-dom';
 
+describe('timeDifference', () => {
```

Comment:
Great test coverage! I'm glad it includes checks for undefined and null timestamp values.